code-studio/src/css/plc/content_creation

```diff
@@ -18,6 +18,7 @@
 
     input, select {
       margin: auto;
+      width: 90%;
```

Comment:
Hmm - looks like when I created the scss files in the plc dir I didn't put extensions. I can do that